src/components/TextInput/TextInput.unit.spec.jsx

```diff
@@ -239,6 +239,61 @@ describe('<TextInput>', () => {
     return check;
   });
 
+  it('passes aXe check when error and ariaDescribedBy present', () => {
```

Comment:
Is it possible to test the inverse of this—that it fails the aXe check without the `ariaDescribedBy`?

Reply:
I don't believe there is really anything programmatically mandatory about associating some element to another via the aria-describedby attribute. The element we tie in could be any random element.